src/VisualStudio/Core/Def/Implementation/TableDataSource/Suppression/SuppressionStateColumnFilterDefinition.cs

```diff
@@ -1,23 +0,0 @@
-﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
The existence of this type is causing the ErrorListPackage to fail to load.  I'm told it's redundant, but still waiting to hear if that is true for VS 2015 and VS "15".